src/main/scala/kamon/newrelic/metrics/NewRelicMetricsReporter.scala

```diff
@@ -62,13 +70,11 @@ object NewRelicMetricsReporter {
       new NewRelicMetricsReporter()
   }
 
-  def buildSender(): MetricBatchSender = {
-    val config = Kamon.config();
+  def buildSender(config: Config = Kamon.config()): MetricBatchSender = {
     val nrConfig = config.getConfig("kamon.newrelic")
     val nrInsightsInsertKey = nrConfig.getString("nr-insights-insert-key")
     SimpleMetricBatchSender.builder(nrInsightsInsertKey)
       .enableAuditLogging()
-      .secondaryUserAgent("newrelic-kamon-reporter", LibraryVersion.version)
```

Comment:
oh crud. this shouldn't go away. will fix.

Reply:
done

Reply:
sorry I didn't catch that!